cmd/kubeadm/README.md

```diff
@@ -0,0 +1,63 @@
+# Kubernetes Cluster Bootstrap Made Easy
+
+## Usage
+
+### `kubeadm init`
+
+It's usually enough to run `kubeadm init`, but in some case you might like to override the
+default behaviour. The flags used for said purpose are described below.
+
+- `--token=<token>`
+
+  By default, a token is generated, but if you are to automate cluster deployment, you will want to
+  set the token ahead of time. Read the docs for more information on the token format.
+
+- `--api-advertise-addresses=<ips>` (multiple values are allowed by having multiple flag declarations or multiple values separated by comma)
+- `--api-external-dns-names=<domain>` (multiple values are allowed by having multiple flag declarations or multiple values separated by comma)
+
+  By default, `kubeadm` will auto detect IP addresses and use that to generate API server certificates.
+  If you would like to access the API via any external IPs and/or hostnames, which it might not be able
+  to detect, you can use `--api-advertise-addresses` and `--api-external-dns-names` to add multiple
+  different IP addresses and hostnames (DNS).
+
+- `--service-cidr=<cidr>` (default: "100.64.0.0/12")
```

Comment:
It's not a public CIDR, it's so-called ["carrier-grade NAT"](https://en.wikipedia.org/wiki/Carrier-grade_NAT), by using this we have less chance to collide with existing private networks in use, and more options to pick for pod network. Also, we can grab a `/12` quite safely, which is the maximum API server supports (for whatever reason that is).